zmon_agent/discovery/kubernetes/cluster.py

```diff
@@ -36,6 +36,7 @@ def __init__(self, region, infrastructure_account):
         # TODO: get config path from ENV variable
         self.namespace = os.environ.get('ZMON_AGENT_KUBERNETES_NAMESPACE')
         self.cluster_id = os.environ.get('ZMON_AGENT_KUBERNETES_CLUSTER_ID')
+        self.alias = os.environ.get('ZMON_AGENT_KUBERNETES_CLUSTER_ALIAS')
```

Comment:
Is the ``alias`` required? If not, then this will lead to ``None`` in entities, may be it would be better to set default value (empty str or cluster_id)?